sport/app/football/conf/context.scala

```diff
@@ -46,6 +46,11 @@ class FootballLifecycle(
       competitionsService.refreshMatchDay(defaultClock)
     }
 
+    jobs.schedule("MaybeMatchDayAgentRefreshJob", "30 * * ? * * *") {
```

Comment:
I notice the question marks on this job schedule don't match the ones on the existing match day refresh? The Quartz cron format is not the same as the standard unix cron, so it's worth double checking this is right!

Reply:
I wonder whether it would be simpler to have one matchday refresh, that conditionally refreshes more regularly. This would be instead of separately running one conditional (and regular) job at the same time as one consistent (and slow) job in tandem. Apart from the complexity of having two jobs that do mostly the same thing, it means that every 5 minutes we kick off duplicate API calls.

As we talked about, this would be a bit easier to manage with Streams where we can hold onto some state from the previous runs. Right now, each job invocation knows nothing else so I can see that in the case where the job runs and no matches are currently running it would be tricky to decide whether or not to do a refresh.

Something to consider!

Reply:
You're right. I changed the cron schedule to be `0 0/1 * * * ?` which is now translated to 
`At second :00, every minute starting at minute :00, of every hour`

Regarding the removing the old job `MatchDayAgentRefreshJob` that runs every 5 minute, I removed it for now. But as you mentioned, it's best if we use Streams to be able to hold on to the previous state. We are right now exploring the usage of Monix and will probably add the implementation to this PR soon.

Reply:
As already discussed in the call, the old job was brought back in